packages/@lwc/ssr-runtime/src/index.ts

```diff
@@ -172,20 +172,29 @@ export class LightningElement implements PropsAvailableAtConstruction {
         return (this.__classList = new ClassList(this));
     }
 
-    getAttribute(attrName: string): string | null {
-        return this.__attrs[attrName] ?? null;
+    #setAttribute(attrName: string, attrValue: string | null): void {
```

Comment:
Can we use private fields? Maybe for now call this `__setAttribute` and re-use that TODO ticket. We can check support/perf later.